lib/experiment/factory.rb

```diff
@@ -9,7 +9,9 @@ module AmplitudeExperiment
   # @param [String] api_key The environment API Key
   # @param [Config] config Optional Config.
   def self.init(api_key, config = nil)
-    @instances.store(@default_instance, Client.new(api_key, config)) unless @instances.key?(@default_instance)
+    unless @instances.key?(@default_instance)
```

Comment:
factory method for local?

Reply:
The factory is for the remote. It has lint error and cannot pass the test. So I fix it.